security/x509.go

```diff
@@ -37,6 +37,8 @@ import (
 // Most fields and settings are hard-coded. TODO(marc): allow customization.
 
 const (
+	// Make certs valid a day before to handle clock issues (specifically boot2docker).
```

Comment:
please link to the specific issue https://github.com/boot2docker/boot2docker/issues/69